### ISSUE TITLE
macOS support

### ISSUE THREAD

**Issue by mtibben** [gballet/go-libpcsclite#3]
I'm trying to [use this library](https://github.com/yawn/ykoath/pull/9) on macOS, but just realised you deactivated support in #2.

Unfortunately I can't find any go library for CryptoTokenKit, so it might be easiest to get this library working. I think it should still be possible to use the PCSC API here. For example, you can see the testpcsc binary on macOS doing it [here](https://opensource.apple.com/source/SmartCardServices/SmartCardServices-55111/src/PCSC/testpcsc.c), which looks like it might be using a socket available at [`/var/run/pcscd.comm`](https://opensource.apple.com/source/SmartCardServices/SmartCardServices-55111/src/PCSC/pcsclite.h) ?

**Comment by gballet** [comment]
Indeed, I naively thought macos would behave like FreeBSD, but it didn't so I deactivated it.

I have tried using this library on someone's mac, and indeed they have their own, incompatible version of PCSC that nonetheless gets a hold of `pcscd.comm`, so unless the user is willing to deactivate the default service, it won't work. I'm happy for any PR that would interface with the crypto token kit, though.
